compiler/backend/src/org/jetbrains/kotlin/codegen/inline/inlineCodegenUtils.kt

```diff
@@ -310,6 +311,22 @@ internal fun firstLabelInChain(node: LabelNode): LabelNode {
     return curNode
 }
 
+internal val MethodVisitor.bytecode: String?
```

Comment:
There is already smae function in FunctionCodegen.renderByteCodeIfAvailable

Reply:
Exactly. During debugging (line number table difference between old and new backend), I somehow thought `FunctionCodegetn.renderByteCodeIfAvailable(anyMethodVisitir)` is not resolvable from new backend, but just tried it again and it works fine. Not sure what I saw, hm. Removed.